src/software/proto/sensor_msg.proto

```diff
@@ -1,12 +1,12 @@
 syntax = "proto3";
 
-import "shared/proto/tbots_robot_msg.proto";
+import "shared/proto/tbots_robot_status_msg.proto";
 import "software/proto/ssl_referee.proto";
 import "software/proto/messages_robocup_ssl_wrapper.proto";
 
 message SensorMsg
 {
-    SSL_Referee ssl_refbox_msg              = 1;
-    SSL_WrapperPacket ssl_vision_msg        = 2;
-    repeated TbotsRobotMsg tbots_robot_msgs = 3;
+    SSL_Referee ssl_refbox_msg                           = 1;
+    SSL_WrapperPacket ssl_vision_msg                     = 2;
+    repeated TbotsRobotStatusMsg tbots_robot_status_msgs = 3;
```

Comment:
Hmm, so we don't have the `Tbots` prefix on any of our other message types. Any particular reason we think we should keep it on `TbotsRobotStatusMsg`

Reply:
I think we should differentiate between messages we define and messages that ssl define

Reply:
So we might want to have the `Tbots` prefix, or a `tbots` namespace, but either way, I think that's outside of the scope of this PR. I think we should file a ticket to update it in the future and change this to be consistent with what is in the code now.

Reply:
I'm not sure exactly what you want to see, so I'm not sure what to put in this ticket

Reply:
Sorry, I think we should name this `RobotStatusMsg` for now to be consistent with other messages (ie. no prefix) and file a ticket to consider adding the `Tbots` prefix or putting all messages in a `tbots` namespace.

Reply:
ah I see, done